mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td

```diff
@@ -1611,7 +1651,7 @@ def Tosa_ReverseOp: Tosa_Op<"reverse", [
 
   let hasFolder = 1;
   let hasVerifier = 1;
-  
```

Comment:
nit: can you remove these whitespace changes as they aren't relevant to this change. Same below for the 'cast' changes.